### DISCUSSION TITLE
When migrate from Deprecated AAD Pod Identity

### DISCUSSION THREAD

**Discussion by BGajda13** [Azure/azure-workload-identity#1304] [Q&A] [answered]
Hello,

I wonder if there is an urgent need to move on from the old Pod Identity approach. We have k8s cluster and dotnet apps based on AAD Pod Identity. Is it needed to do it as soon as possible?

**Comment by aramase** [comment, marked as answer]
Hello 👋🏻  AAD Pod Identity has also been archived and is no longer maintained (xref: https://github.com/Azure/aad-pod-identity?tab=readme-ov-file#-announcement). The repo is only available in read-only mode and there are no releases since last year. You should be using Azure Workload Identity already.
